scripts/createLocalPackage.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Test if there are currently untracked files and notify the user they exist (do not move things around for the user)
+FILES_TO_CLEAN=$(git clean -n)
+if [ ! -z "$FILES_TO_CLEAN" ]
+then
+    echo "Please note that a byproduct is deleting ALL untracked files from your folder, these files will be deleted:"
+    echo "${FILES_TO_CLEAN}"
+    select yn in "Yes" "No"; do
+        case $yn in
+            Yes ) break;;
+            No ) exit;;
+        esac
+    done
+fi
+
+PACKAGE_JSON=$(git diff package.json)
+if [ ! -z "$PACKAGE_JSON" ]
+then
+    echo "Your changes to pacakge.json will be erased, continue?"
+    select yn in "Yes" "No"; do
+        case $yn in
+            Yes ) break;;
+            No ) exit;;
+        esac
+    done
+fi
+
+# Delete previous tar
+rm -f react-native-ui-lib.tgz
+# Build javascript files
+npm run build
+# Package into a tar
+npm pack
+# Clean ALL untracked files
+git clean -f
+git checkout package.json
+# Rename tar
+mv react-native-ui-lib-5.0.0.tgz react-native-ui-lib.tgz
```

Comment:
Maybe mv to the user Downloads folder?

Reply:
Good idea, this should probably remove the need for the ignore, and solve the above comment.